pkg/operator/csidriveroperator/deploymentcontroller.go

```diff
@@ -25,8 +27,6 @@ import (
 // It replace ${LOG_LEVEL} in the Deployment with current log level.
 // It replaces images in the Deployment using  CSIOperatorConfig.ImageReplacer.
 // It produces following Conditions:
-// <CSI driver name>CSIDriverOperatorDeploymentAvailable
-// <CSI driver name>CSIDriverOperatorDeploymentProgressing
```

Comment:
@jsafrane currently CSO sets `Progressing=True` if it doesn't find any `*Progressing` condition:

https://github.com/bertinatto/cluster-storage-operator/blob/74c8159c8b6b1d616a4da3fdab1801e3e14f36c6/pkg/operator/csidriveroperator/crcontroller.go#L205-L213